day_7/checker_board.rb

```diff
@@ -0,0 +1,39 @@
+# define a function that take a width
+
+#every other space should be an x or a ' '
+
+#if previous line statred with x, start new line with ' '
+
+#puts number of times specified by user
+#note
+
+puts "How many spaces heigh would you like your board to be?"
+puts "> "
+height = $stdin.gets.chomp.to_i
+puts "How many spaces across would you like your board to be?"
+puts "> "
+width = $stdin.gets.chomp.to_i
+
+
+
+def counter(h, w)
+  h /= 2
+  w /= 2
+
+  while h > 0
+    w.times do
+      print "X"
+      print " "
+    end
+    puts " "
+    print " "
+    w.times do
+      print "X"
+      print " "
+    end
+    puts " "
```

Comment:
Really nice work with programming this out. My boards came out 1 unit too short when running the code, but otherwise it works really well. Great work!
It looks like you're using print and puts statements to build out strings character by character for each line. A more conventional approach would be to build the full string (either of one line or the entire board) and then print/puts it out.